outline/data_structures.md

```diff
@@ -289,24 +289,26 @@ be confusing.
 > After the creation, we want to save a new value associated to the
 > key. The `assoc` function can be used by assigning a new value to
 > the existing key.
-> Also, there's handy function `update-in`. The function takes map and
-> a key or list of keys with a function. The value of specified key will be the
-> first argument of the given function.
+> Also, there's handy function `update`. The function takes map and
+> a key with a function. The value of specified key will be the first
+> argument of the given function.
```

Comment:
These notes are mainly speaker/TA notes. The more important part is the changes to the examples.